app/controllers/unattended_controller.rb

```diff
@@ -56,6 +56,12 @@ def require_ssl?
 
   private
 
+  def render_custom_error(status, error_message, params)
+    logger.error error_message % params
+    # add a comment character (works with Red Hat and Debian systems) to avoid parsing errors
+    render(:text => '# ' + _(error_message) % params, :status => status)
```

Comment:
missing content-type? It's returned as text/html and formatted differently to the template itself.